app/controllers/forms.scala

```diff
@@ -381,9 +437,12 @@ object SubscriptionFieldsForm {
         mapping(
           "name" -> text,
           "description" -> text,
+          "sortDescription" -> text,
```

Comment:
I think this is a type!
sortDescription
shortDescription

Reply:
:+1: